pages/osx/fileicon.md

```diff
@@ -0,0 +1,20 @@
+# fileicon
+
+> a macOS CLI for managing custom icons for files and folders, as a programmatic alternative to interactively using Finder.
+> More information: <https://github.com/mklement0/fileicon>.
+
+- Assign custom icon derived from image file 'img.png' to file 'foo':
+
+`fileicon set foo img.png`
```

Comment:
```suggestion
- Set a custom icon for a specific file or directory:

`fileicon set {{path/to/file_or_directory}} {{path/to/icon.png}}`
```
Does this command only accept PNG icons?

Reply:
It probably takes all picture files, the documentation is no help since it uses .png as well... however it does suggest it can be changed to foo (without file extension) however this is probably redundant

Reply:
Im going to leave it as png, the user will probably ignore it and it just gives an example picture, and if it does only accept .pngs, even better.

Its a win-win

Reply:
@CleanMachine1 why did you resolve? I'd say my suggestion is still valid.

Reply:
Yes, @marchersimon's suggestion would be better than the current one.

Reply:
I thought I added it. I completely agree yours is better